spec/features/posting_a_peep_spec.rb

```diff
@@ -0,0 +1,10 @@
+feature 'Posting a peep' do
+  scenario 'A user can post a peep to Chitter' do
+    visit('/peep')
+    fill_in('message', with: 'Example Peep')
+    click_button('Submit')
+
+    expect(page).to have_content 'Example Peep'
+    expect(page).to have_content '2021-06-25'
```

Comment:
What happens if I run this test today?

Reply:
It seems the test is brittle because it's linked to a specific date (as per when the test was written). To avoid that, a suggestion would be to create a new time object and then format it to get the date, for example:

```ruby
time_today = Time.now
date_today = time_today.strftime("%Y-%m-%d")
```

Then you would expect the page to include the date as per today in this feature test.